### DISCUSSION TITLE
Disable Two Factor Authentication without having access to authenticator app

### DISCUSSION THREAD

**Discussion by hassanali542** [community/community#139595] [GitHub Education]
### Hi everyone,

I have applied two factor authentication on my github account but somehow i don't have access to my authenticator application and i am logged out of my account already
is there any way that i can recover my account , i have access to the email i registered.

**Comment by Rishikesh1601** [comment]
To recover your GitHub account if you've enabled two-factor authentication (2FA) but no longer have access to your authenticator app, you can follow these steps:

### 1. **Use Recovery Codes**
   - When you set up 2FA on GitHub, you were provided with a set of recovery codes. These codes can be used to regain access if you lose your authenticator app.
   - If you have the recovery codes saved, follow these steps:
     - Go to [GitHub's login page](https://github.com/login).
     - Enter your username and password.
     - On the 2FA screen, select the option to use a recovery code.
     - Enter one of your recovery codes to log in.

### 2. **Use Backup Authentication Methods**
   - If you set up another 2FA method, such as SMS-based authentication, GitHub will allow you to use it instead of the authenticator app.
   - If SMS backup is available, use that to receive a one-time code via text message to log in.

### 3. **Account Recovery with GitHub Support**
   If you do not have access to recovery codes or any backup 2FA method, you can recover your account by reaching out to GitHub support:
   - Go to the [GitHub Support page](https://support.github.com/request).
   - Choose the **Account recovery** category.
   - Explain your situation and provide as much information as possible (such as your GitHub username and the email associated with the account).
   - GitHub may send you instructions to verify your identity using your registered email.

### 4. **Check for Trusted Devices**
   - If you’ve previously logged into GitHub on a trusted device and haven’t logged out, you might still be able to access your account without 2FA. From there, you can disable or reset 2FA.

### 5. **Use SSH or PAT (Personal Access Token)**
   - If you have SSH keys or a PAT associated with your account, you might be able to use them to access your repositories or certain account settings.
   - If you can access your account using these methods, disable 2FA from the settings and set it up again.

### Recommendations for Future
   - After regaining access, consider saving recovery codes in a safe place and adding multiple 2FA methods (such as SMS backup) to avoid this issue again.

Good luck with the recovery!
